modules/EvseManager/manifest.yaml

```diff
@@ -245,6 +245,12 @@ config:
       - X1
       - F
     default: X1
+  lock_connector_in_state_b:
+    description: >-
+      Indicates if the connector lock should be locked in state B. If set to false, connector will remain unlocked in CP state B
+      Warning: This violates IEC61851-1:2019 D.6.5 Table D.9 line 4 and should not be used in public environments!
```

Comment:
Change to `[...] If set to false, connector will remain unlocked in CP state B and this violates IEC61851-1:2019 D.6.5 Table D.9 line 4 and should not be used in public environments!`